### ISSUE TITLE
Layouts don't update when initially loaded

### ISSUE THREAD

**Issue by Cristian006** [phillfarrugia/swipeable-view-stack#2]
![offset_layout_on_load](https://user-images.githubusercontent.com/14713485/39091467-41a27992-45a9-11e8-9931-f20bcae50ebc.gif)

**Comment by roimulia** [comment]
Did you solved it?

**Comment by Cristian006** [comment]
@roimulia Yeah, if you just want to fix the layout issue, the only code changes you need are here: https://github.com/Cristian006/swipeable-view-stack/commit/fc7c70eb57ccf2610cdb2427e6b28145118eb7cb

**Comment by roimulia** [comment]
Awesome, thank you!
